lib/components/ViewWalletButton/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { cx } from "@emotion/css";
+import { Launch } from "@mui/icons-material";
+import React, { FC } from "react";
+
+import { ButtonPrimary } from "../";
+import { concatAddress } from "../../utils";
+import { ButtonBaseProps } from "../Buttons/ButtonBase";
+import * as styles from "./styles";
+
+type Props = {
+  address: string;
+} & ButtonBaseProps;
+
+export const ViewWalletButton: FC<Props> = (props) => {
+  const className = cx(styles.viewWalletButton, props.className);
+
+  return (
+    <div className={className}>
+      <span>{concatAddress(props.address)}</span>
+      <ButtonPrimary
+        href="https://polygon.tor.us/"
+        target="_blank"
+        rel="noopener noreferrer"
+        label="VIEW"
```

Comment:
```suggestion
        label={t(`View`)}
```

=> for translators

Reply:
it doesn't work, as mentioned above. I think I'm missing something

Reply:
Ah, I see it now.

You created the component in @klimadao/lib
In this repo there is no lingui.

I would do this:

1. move ViewWalletButton component to carbonmark/components (it won't be in use somewhere else anyways)
2. Import ButtonPrimary from carbonmark/components/Buttons into this component
3. try again with `label={<Trans>View</Trans>}`
4. Here it will work

Reply:
works now, thanks